atmo/clusters/tasks.py

```diff
@@ -0,0 +1,12 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+from django.core.management import call_command
+
+
+def delete_clusters():
```

Comment:
From an API point of view, I'd rather see the app logic of the jobs to live here and the management commands to call the Python functions defined here -- both to have a consistent place for the various jobs (and not have to look for the command files) and also to not have `call_command` in the call stack at all